src/Paket.Core/ProjectFile.fs

```diff
@@ -941,6 +941,9 @@ module ProjectFile =
                 project.ProjectNode.AppendChild analyzersNode |> ignore
             )
 
+    let touch (project:ProjectFile) =
+        if File.Exists(project.FileName) then
```

Comment:
It does not change any behavior as before the touching was only exposed indirectly through routines where the file was known to be present.

But I agree, `touch` is typically expected to create the file if not present. Since it is exposed directly now, we better adapt it to follow that expectation.